test/misc.jl

```diff
@@ -0,0 +1,178 @@
+const case_ids = [7, 22,
+                #140, requires structural_simplify fix
+                170, 325,
+                # 679 requires structural_simplify fix
+                ]
+
+# const case_ids = [325]
+const cases = map(x -> x[end-4:end], .*("0000", string.(case_ids)))
+
+const algomap = Dict("00177" => Rodas4(),
```

Comment:
misc is not a great name for this file. what are these testing?

Reply:
It is just a few representative test suite cases that fail, despite the tests in the other files included in runtests.jl pass.

I mean in reactionsystem.jl there are largely unit tests, then there are some that are more integration tests. Perhaps misc.jl could be renamed to functional_tests.jl. What do you think?